Formula/leaf-proxy.rb

```diff
@@ -0,0 +1,65 @@
+class LeafProxy < Formula
+  desc "Lightweight and fast proxy utility"
+  homepage "https://github.com/eycorsican/leaf"
+  url "https://github.com/eycorsican/leaf/archive/v0.2.11.tar.gz"
+  sha256 "1c17c47c86560e8dba24c02b29bba1b13bfd768b810ab536c3a9ea4fb13d4651"
+  license "Apache-2.0"
+  head "https://github.com/eycorsican/leaf.git"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+
+  conflicts_with "leaf", because: "both install `leaf` binaries"
+
+  resource "lwip" do
+    url "https://github.com/eycorsican/lwip-leaf.git",
+        revision: "0c189c0fffa52354738ebba6033d365bc0d096fd"
+  end
+
+  patch :DATA
```

Comment:
This needs to be submitted upstream. New formula will not be admitted with patches.

Reply:
This just disables [stripping](https://doc.rust-lang.org/cargo/reference/unstable.html#profile-strip-option) which is only supported in nightly Rust. It doesn't change the actual code.

Reply:
If upstream doesn't support building with a stable rust version then that is a problem that should be addressed before it can be admitted to homebrew.

Reply:
Upstream have replaced this config with environment flags, so it should build on stable now.